cookiecutter-frontend-app/cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This `json` file provides default values for the template, though the developer can still override when run interactively (the default).
You're prompted for each value and you can leave it blank (just press enter) for the default.